public/js/main.js

```diff
@@ -11,9 +11,23 @@ function toggleNightMode() {
 	localStorage.setItem("night", (night == "true") ? "false" : "true");
 }
 
-// Switches between themes when a new one is selected
-function switchThemes(){
-	themeName = document.getElementById("theme-selector").value
+function changeTheme(opt) {
+	theme = opt.value;
+	localStorage.setItem("theme", theme);
+	document.getElementById("theme").href = "/css/" + theme;
+	console.log(theme);
+}
+
+*/
+
+
+function switchThemes(themeName=null){
```

Comment:
Unfortunately no, I'm running on a virtual machine with a wifi connection and I can't bridge it over to windows because drivers. 
I'll just replace it with the older syntax so I can be sure it works.